### ISSUE TITLE
Printing with the MAP_ONLY layout template fails

### ISSUE THREAD

**Issue by bsnider** [Esri/basic-viewer-template#16]
The export options JSON parameter is missing from the request when printing with any template.  Other templates still successfully print, but if the MAP_ONLY template is chosen (by first adding it as an option in the printLayouts in the index.html file), the request will fail with the error: 
ERROR 001305: Missing or invalid 'exportOptions' property in WebMap; cannot derive a size for the output image. Failed to execute (Export Web Map). Failed to execute (Export Web Map Task).

The attempted remedy of hard-coding the exportOptions parameter either in the index.html file or in the layout.js file (around line 1000) also fails:

exportOptions = {
            'outputSize': [
                800,
                1100
            ],
            "dpi": 96
        }

The JSON request adds these parameters, but the outputSize dimensions are set to null, resulting in the same error message:

{
    "exportOptions": {
        "outputSize": [
            null,
            null
        ],
        "dpi": 96
    }
}

To fix the JSON request, the null values can be changed to [800, 1100]:

{
    "exportOptions": {
        "outputSize": [
            800,
            1100
        ],
        "dpi": 96
    }
}

**Comment by jgravois** [comment]
isn't the MAP_ONLY template already added [here](https://github.com/Esri/basic-viewer-template/blob/master/javascript/layout.js#L1034)?

**Comment by bsnider** [comment]
No, those steps were attempted, but the exportParameters are still missing.

**Comment by jgravois** [comment]
i can't repro the behavior you're describing.  with most current build, MAP_ONLY print requests include export options and output appropriately.

in index.html, i made only the changes below:
```
printlayout: true
proxyurl: 'valid proxy path'
```
in running app, click print dropdown and choose `MAP_ONLY`
